INSTALL.md

```diff
@@ -70,7 +70,7 @@ Replace kci_storage_fqdn with the name you'll be using for the storage.
 
 * `hosts`:
 Add the TARGET_MACHINE and configure how to ansible should connect, for example if your machine
-is named backend.mydomain.local with IP 10.0.17.2 and you connect directly as root (using a ssh key),
+is named server.mydomain.local with IP 10.0.17.2 and you connect directly as root (using a ssh key),
```

Comment:
it is important that we keep the domain name used for TARGET_MACHINE consistent throughout the file. Even if they are just examples

Reply:
Erm, yes I think there's some confusion around `TARGET_MACHINE` and `TARGET_NAME`.  I would need to read the whole file again to check all inconsistencies.